File: 0001 Code Chef/0019 Water Consumption/implementation/0001 c/water_consumption.c
#include <stdio.h>

int main(){
    int test;
    scanf("%d",&test);
    while(test > 0){
        int water;
        scanf("%d",&water);
        if(water >= 2000){
            printf("YES\n");
        }else{
            printf("NO\n");
        }
        test = test - 1;
    }
    return 0;
}

File: 0001 Code Chef/0019 Water Consumption/implementation/0004 java/WaterConsumption.java
import java.util.Scanner;

public class WaterConsumption{
    public static void main(String[] args){
        Scanner console = new Scanner(System.in);
        String stringInput;

        stringInput = console.nextLine();
        int test = Integer.parseInt(stringInput);
        while(test > 0){
            stringInput = console.nextLine();
            int water = Integer.parseInt(stringInput);
            if(water >= 2000){
                System.out.println("YES");
            }else{
                System.out.println("NO");
            }
            test = test - 1;
        }

    }
}